java/src/org/openqa/selenium/chromium/AddHasNetworkConditions.java

```diff
@@ -0,0 +1,99 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.chromium;
+
+import com.google.auto.service.AutoService;
+import com.google.common.collect.ImmutableMap;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.internal.Require;
+import org.openqa.selenium.remote.AdditionalHttpCommands;
+import org.openqa.selenium.remote.AugmenterProvider;
+import org.openqa.selenium.remote.CommandInfo;
+import org.openqa.selenium.remote.ExecuteMethod;
+import org.openqa.selenium.remote.http.HttpMethod;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Predicate;
+
+import static org.openqa.selenium.remote.BrowserType.CHROME;
+import static org.openqa.selenium.remote.BrowserType.EDGE;
+
+@AutoService({AdditionalHttpCommands.class, AugmenterProvider.class})
+public class AddHasNetworkConditions implements AugmenterProvider<HasNetworkConditions>, AdditionalHttpCommands {
+
+  public static final String GET_NETWORK_CONDITIONS = "getNetworkConditions";
+  public static final String SET_NETWORK_CONDITIONS = "setNetworkConditions";
+  public static final String DELETE_NETWORK_CONDITIONS = "deleteNetworkConditions";
+
+
+  private static final Map<String, CommandInfo> COMMANDS = ImmutableMap.of(
+    GET_NETWORK_CONDITIONS, new CommandInfo("/session/:sessionId/chromium/network_conditions",HttpMethod.GET),
+    SET_NETWORK_CONDITIONS, new CommandInfo("/session/:sessionId/chromium/network_conditions",HttpMethod.POST),
+    DELETE_NETWORK_CONDITIONS, new CommandInfo("/session/:sessionId/chromium/network_conditions", HttpMethod.DELETE));
+
+  @Override
+  public Map<String, CommandInfo> getAdditionalCommands() {
+    return COMMANDS;
+  }
+
+  @Override
+  public Predicate<Capabilities> isApplicable() {
+    String[] validBrowsers = new String[] { EDGE, CHROME, "msedge" };
```

Comment:
I've seen this list before... Perhaps a constant on `ChromiumDriver` itself? `KNOWN_CHROMIUM_BROWSERS`?